### PR TITLE
Restructure Compass docs

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Restructure the docs
- Update README.md files

**Related issue(s)**

### CODE REVIEW

File: docs/archive/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Archive
+
+## Overview
+
+This directory contains the following archived documents:
```
Comment (kazydek) [inline docs/archive/README.md]:
```suggestion
This directory contains the following archived document:
```

File: docs/investigations/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Investigations
+
+## Overview
+
+This directory contains the following investigations:
+
+- [Storage - Requirements](./investigations/storage/requirements.md)
```
Comment (kazydek) [inline docs/investigations/README.md]:
```suggestion
- [Storage - Requirements](./storage/requirements.md)
```

Comment (kazydek) [reply docs/investigations/README.md]: Here and everywhere below

Comment (klaudiagrz) [reply docs/investigations/README.md]: Good point, thanks!

**Pull request merged**